introduction.md

```diff
@@ -111,10 +113,15 @@ The following changes and additions have been made to update the 2013 WCAG2ICT d
         * [style property](#dfn-style-properties)
         * [target](#dfn-targets)
         * [up event](#dfn-up-event)
-* Updated terms
+* Updated terms:
     * [large scale](#dfn-large-scale)
     * [set of web pages](#dfn-set-of-web-pages)
     * [set of non-web documents](#set-of-documents)
     * [set of software programs](#set-of-software-programs)
-* Updated sections
-    <div class=note">In this Note, most of the existing sections have undergone WCAG2ICT Task Force review and updates. Many sections needed only minor editorial and link URL updates, such as the guidance for the WCAG 2.0 success criteria.</div>
+* Updated sections:
+    <div class="note">
+      <ul>
+        <li>Most of the existing sections have undergone WCAG2ICT Task Force review and updates.</li>
```

Comment:
Updated the text for accuracy. It was not quite right even before the suggested edits.
```suggestion
        <li>The existing sections have all undergone WCAG2ICT Task Force review to identify updates needed.</li>
```